templates/cartridge/deps.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/sh
```

Comment:
May be we should remove `deps.sh` and use hooks instead?

Reply:
This is reasonable. But then we need to also change the getting started example.

Reply:
This script is used to install dev dependencies, such as luatest and luacheck. I think that it shouldn’t be removed, it allows to fix dev dependencies versions in application template.
But I agree that getting-started example should be changed to use a new build command.

Reply:
Let's do it in separate PR